Com.OneSignal.Android/SendOutcomeEventSuccessHandler.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Collections.Generic;
+using Com.OneSignal.Abstractions;
+using Com.OneSignal.Android;
+
+namespace Com.OneSignal
+{
+   public class SendOutcomeEventSuccessHandler : Java.Lang.Object, Android.OneSignal.IOutcomeCallback
+   {
+    
+      readonly SendOutcomeEventSuccess _sendOutcomeEventSuccess;
+
+	   public SendOutcomeEventSuccessHandler(SendOutcomeEventSuccess sendOutcomeEventSuccess) => _sendOutcomeEventSuccess = sendOutcomeEventSuccess;
+
+      public void OnSuccess(OutcomeEvent outcome)
+      {
+         OSOutcomeEvent outcomeEvent = OSSendOutcomeEventSuccessToNative(outcome);
+         _sendOutcomeEventSuccess(outcomeEvent);
+      }
+
+      private static OSOutcomeEvent OSSendOutcomeEventSuccessToNative(OutcomeEvent outcome)
```

Comment:
This logic is repeated in `Com.OneSignal.Abstractions/OSOutcomeEvent.cs` as well. Can we reuse this outcome dictionary parsing between the two?